charts/consul/test/acceptance/tests/partitions/partitions_test.go

```diff
@@ -0,0 +1,148 @@
+package partitions
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/consul"
+	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/environment"
+	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/helpers"
+	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/k8s"
+	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/logger"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Test that Connect works in a default installation.
+// i.e. without ACLs because TLS is required for setting up Admin Partitions.
+func TestMeshGatewayDefault(t *testing.T) {
+	env := suite.Environment()
+	cfg := suite.Config()
+
+	if !cfg.EnableEnterprise {
+		t.Skipf("skipping this test because -enable-enterprise is not set")
+	}
+
+	if !cfg.UseKind {
+		t.Skipf("skipping this test because Admin Partition tests are only supported in Kind for now")
+	}
+
+	primaryContext := env.DefaultContext(t)
+	secondaryContext := env.Context(t, environment.SecondaryContextName)
+
+	ctx := context.Background()
+
+	primaryHelmValues := map[string]string{
+		"global.datacenter": "dc1",
+		"global.image":      "hashicorp/consul-enterprise:1.11.0-ent-alpha",
+
+		"global.adminPartitions.enabled": "true",
+		"global.enableConsulNamespaces":  "true",
+		"global.tls.enabled":             "true",
+
+		"server.exposeGossipAndRPCPorts": "true",
+
+		"connectInject.enabled": "true",
+		"controller.enabled":    "true",
+	}
+
+	if cfg.UseKind {
+		primaryHelmValues["global.adminPartitions.service.type"] = "NodePort"
+		primaryHelmValues["global.adminPartitions.service.nodePort.https"] = "30000"
+	}
+
+	releaseName := helpers.RandomName()
+
+	// Install the primary consul cluster in the default kubernetes context
```

Comment:
nit: Should we use other names for the clusters, i.e. not "primary"
```suggestion
	// Install the primary consul cluster in the default kubernetes context.
```

Also another nit: I think other go comments need to be updated (it's probably my fault since I wrote the mesh gw tests)

Reply:
you shouldn't take the blame for my lazy copy-pasting lol.

Reply:
lol you are too kind